study-builder/studies/osteo/subs.conf

```diff
@@ -8,4 +8,10 @@
     "facebook": "https://www.facebook.com/osteosarcomaproject/",
     "instagram": "https://www.instagram.com/the_osproject/"
   }
+  "id": {
+    "study": "CMI-OSTEO"
```

Comment:
Gets used in Family history config files. Copied this id/study thing from testboston.